api/tests/test_gdpr/gdpr_key.py

```diff
@@ -0,0 +1,68 @@
+"""
+ This file is copied from: https://github.com/City-of-Helsinki/helsinki-profile-gdpr-api/blob/main/tests/keys.py
+ <3
+"""
+
+
+from jose import jwk
+from jose.constants import ALGORITHMS
+
+
+def _build_key(private_pem, public_pem):
+    class _Key:
+        pass
+
+    key = _Key()
+    key.jose_algorithm = ALGORITHMS.RS256
+    key.private_key_pem = private_pem
+    key.public_key_pem = public_pem
+    key.public_key_jwk = jwk.construct(public_pem, key.jose_algorithm).to_dict()
+
+    # Ensure values are strings and not bytes
+    for name in ["n", "e"]:
+        value = key.public_key_jwk[name]
+        if isinstance(value, bytes):
+            key.public_key_jwk[name] = value.decode("utf-8")
+
+    return key
+
+
+rsa_key = _build_key(
+    """-----BEGIN PRIVATE KEY-----
```

Comment:
Ahaa! This is a test file. Good that we are not leaking any sensitive data.

Reply:
Yes, it's a copy from [helsinki-profile-gdpr-api](https://github.com/City-of-Helsinki/helsinki-profile-gdpr-api/blob/main/tests/keys.py) tests. Parking permits seems to use the same as well.